src/wirecloud/commons/static/js/StyledElements/Accordion.js

```diff
@@ -46,7 +46,7 @@
         var expander;
 
         options = utils.merge({
-            'listenOnTitle': true
+            'listenOnHeader': true
```

Comment:
Didn't find any usage of this element but in any case I think it would also be a good change for accordions